tests/dialects/test_mpi.py

```diff
@@ -1,67 +1,6 @@
-from conftest import assert_print_op
```

Comment:
Are there tests for the new ops?

Reply:
The tests were added in 52343c829, except for `mpi.comm_size`. I can add an example for that as well though!

Reply:
Hmm yeah, I guess I was blind when reading the PR...